### PR TITLE
add-data URL fixes

### PR DESCRIPTION
**Pull request by FilipLeitner**

### CODE REVIEW

File: projects/hslayers/src/components/add-data/url/arcgis/add-data-url-arcgis.service.ts [hunk 1]
```diff
@@ -17,28 +18,63 @@ import {getPreferedFormat} from '../../../../common/format-utils';
...
     public hsArcgisGetCapabilitiesService: HsArcgisGetCapabilitiesService,
     public hsDimensionService: HsDimensionService,
     public hsLayoutService: HsLayoutService,
     public hsMapService: HsMapService,
-    public hsUtilsService: HsUtilsService
+    public hsUtilsService: HsUtilsService,
+    public HsEventBusService: HsEventBusService,
+    public HsAddDataUrlService: HsAddDataUrlService
   ) {
+    console.log('arcgis');
```
Comment (raitisbe) [inline projects/hslayers/src/components/add-data/url/arcgis/add-data-url-arcgis.service.ts:34]: Probably needs to be removed

**Review by raitisbe — APPROVED** [review]

File: projects/hslayers/src/components/add-data/url/add-data-url.service.ts [hunk 2]
```diff
@@ -33,4 +35,48 @@ export class HsAddDataUrlService {
...
+  ): void {
+    if (!layerToSelect) {
+      return;
+    }
+    for (const service of services) {
+      if (service.Layer) {
+        for (const layer of service.Layer) {
+          if (layer[selector] == layerToSelect) {
+            layer.checked = true;
+            setTimeout(() => {
```
Comment (raitisbe) [inline projects/hslayers/src/components/add-data/url/add-data-url.service.ts:58]: Extract to function maybe to remove the duplication.

Comment (FilipLeitner) [reply projects/hslayers/src/components/add-data/url/add-data-url.service.ts:58]: good point, thanks

File: projects/hslayers/src/components/add-data/url/add-data-url.service.ts [hunk 3]
```diff
@@ -55,28 +55,26 @@ export class HsAddDataUrlService {
...
           if (layer[selector] == layerToSelect) {
             layer.checked = true;
-            setTimeout(() => {
-              const id = `#hs-add-layer-${layer[selector]}`;
-              const el = this.HsLayoutService.contentWrapper.querySelector(id);
-              if (el) {
-                el.scrollIntoView();
-              }
-            }, 1000);
+            this.scrollToLayer(service[selector]);
```
Comment (raitisbe) [inline projects/hslayers/src/components/add-data/url/add-data-url.service.ts:58]: Previously you had layer[selector] not {service[selector] here. Is it ok?

Comment (FilipLeitner) [reply projects/hslayers/src/components/add-data/url/add-data-url.service.ts]: Definately no, thanks once more

**Pull request merged**